BUILD

```diff
@@ -292,3 +314,19 @@ upgrading GCC. If you have libstdc++ corresponding to GCC 5.x or 6.x, you *must*
 old ABI, but Dinit will be broken if you upgrade to GCC 7. If you have libstdc++ from GCC 7, you
 *must* build with the new ABI. If the wrong ABI is used, Dinit may still run successfully but any
 attempt to load a non-existing service, for example, will cause Dinit to crash.
+
+
+Special note for Clang (on systems using Libcxxrt C++ runtime)
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+
+Clang on platforms that also use libcxxrt as the C++ runtime library has an issue that breaks
+exception handling and may prevent Dinit from working correctly when compiled with the "-fno-rtti"
+compiler option. The known platforms exhibiting this problem are FreeBSD and macOS.
+See "Recommended compiler options", above, for information about the "-fno-rtti" compiler option.
+
+Details regarding the issue with Clang can be found here:
+
+    https://github.com/llvm/llvm-project/issues/66117
+
+It's recommended not to use the "-fno-rtti" compiler option on the mentioned platforms. As a
+result, this makes the output binary bigger.
```

Comment:
Prefer "larger" to "bigger".

"As a result, this makes" - No, either "As a result, the output binary will be larger" or "This will make the output binary larger".